tests/quick_tests/kokkos.cc

```diff
@@ -0,0 +1,40 @@
+
+#include <deal.II/base/config.h>
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
+#include <Kokkos_Core.hpp>
+DEAL_II_ENABLE_EXTRA_DIAGNOSTICS
+
+struct FillFunctor
+{
+  KOKKOS_FUNCTION void
+  operator()(const int i) const
+  {
+    m_values(i) = i;
+  }
+  Kokkos::View<int *> m_values;
+};
+
+struct SumFunctor
+{
+  KOKKOS_FUNCTION void
+  operator()(const int i, int &sum) const
+  {
+    sum += m_values(i);
+  }
+  Kokkos::View<int *> m_values;
+};
```

Comment:
Should the quicktest use the host memory? The current test will fail if you are on a login node without GPU. Though it might also be interesting to know that the device set up works correctly...

Reply:
My thinking was that this should verify that we can run with the final configuration on the target node. Let's see if someone else has an opinion.